src/test/java/com/rockwell/basics/animals/LionTest.java

```diff
@@ -0,0 +1,41 @@
+package com.rockwell.basics.animals;
+
+import com.rockwell.basics.animals.mammal.Lion;
+import org.junit.Test;
+
+import static junit.framework.Assert.assertEquals;
+
+public class LionTest {
+
+    Lion lion = new Lion();
+
+    @Test
+    public void shouldWeightAsMuchAsEat() {
+        //given
```

Comment:
Correct approach. Can you explain why did you use Given, When, Then expression here? It is correct but what is behind of this decision?